MSAL/src/MSALPublicClientApplication.m

```diff
@@ -268,7 +268,14 @@ - (id)initWithClientId:(NSString *)clientId
     MSALPublicClientApplicationConfig *config = [[MSALPublicClientApplicationConfig alloc] initWithClientId:clientId redirectUri:redirectUri authority:authority];
     
 #if TARGET_OS_IPHONE
-    config.cacheConfig.keychainSharingGroup = keychainGroup ?: [[NSBundle mainBundle] bundleIdentifier];
+    if (keychainGroup)
+    {
+        config.cacheConfig.keychainSharingGroup = keychainGroup;
+    }
+    else
+    {
+        config.cacheConfig.enableKeychainSharing = NO;
```

Comment:
nit: there's no need to set it, because cache config itself checks bundle id and returns correct boolean, right?